src/pages/documentation/adobe-analytics-mobile-services/release-notes.md

```diff
@@ -1,27 +0,0 @@
-# Release Notes
-
-Release notes and change logs for the Adobe Mobile Services extension
-
-## November 5, 2021
-
-### iOS AEPMobileServices 3.0.3
```

Comment:
can we preserve these in previous versions?

Reply:
IIRC - I kept them in the previous versions, but I'll double check and let you know if I did not.

Reply:
I did not! I'll go re-add them.

Reply:
Or, sorry - I preserved it within the Mobile Services folder, but not in the release-notes folder. So I'll add those there.